src/solver/simulation/remix-storage/shave-peaks-by-remix-storage-gen.cpp

```diff
@@ -64,6 +78,7 @@ void shavePeaksByRemixingStorageGen(const std::vector<double>& Load,
 
         if (!exchange.isPossible())
         {
+            collectDebugInfos((*cyclic_it).get(), debugStream);
```

Comment:
So, according to the current code, what we want is : collect debug information about the current storage (in the loop) **only if** the exchange of production between 2 hours is **not** possible ?
It seems strange to me.